README.md

```diff
@@ -1,4 +1,5 @@
-Jeevan's collection of simple Bash scripts.
+
+Mano's collection of simple Bash scripts.
```

Comment:
@Prudhviraju-E  Please change this to Jeevan

Reply:
Already explained the process.I think it is resolved. @mxp737 @djs167